fabric-rendering-v1/src/client/java/net/fabricmc/fabric/mixin/client/rendering/InGameHudMixin.java

```diff
@@ -26,11 +28,89 @@
 import net.minecraft.client.render.RenderTickCounter;
 
 import net.fabricmc.fabric.api.client.rendering.v1.HudRenderCallback;
+import net.fabricmc.fabric.api.client.rendering.v1.hud.VanillaHudElements;
+import net.fabricmc.fabric.impl.client.rendering.hud.HudElementRegistryImpl;
 
 @Mixin(InGameHud.class)
-public class InGameHudMixin {
+public abstract class InGameHudMixin {
```

Comment:
```suggestion
abstract class InGameHudMixin {
```
This might as well be fixed to be package-private since the modifiers are changing anyway.